projects/core/src/cart/facade/active-cart.service.ts

```diff
@@ -220,15 +221,17 @@ export class ActiveCartService implements OnDestroy {
     );
   }
 
-  private loadOrMerge(
+  /**
+   * Loads cart or upon login, whenever there's an existing cart, merge it into the current user cart
+   * cartId will be defined (not '', null, undefined)
+   */
+  protected loadOrMerge(
     cartId: string,
     userId: string,
     previousUserId: string
   ): void {
-    // for login user, whenever there's an existing cart, we will load the user
-    // current cart and merge it into the existing cart
-    // cartId will be defined (not '', null, undefined)
     if (cartId === OCC_CART_ID_CURRENT) {
+      // Load cart if it is current
```

Comment:
I don't know if this comment brings any value ;)

Reply:
Comment removed